bitnami/node/README.md

```diff
@@ -186,34 +162,34 @@ ingress:
         - example.com
 ```
 
-## Connect your application to an already existing database
+### Connect your application to an already existing database
 
-1. Create a secret containing your database credentials:
+1. Create a secret containing your database credentials (named `my-database-secret` as example), you can use the following options:
 
   ```console
-  $ kubectl create secret generic my-database-secret --from-literal=host=YOUR_DATABASE_HOST --from-literal=port=YOUR_DATABASE_PORT --from-literal=username=YOUR_DATABASE_USER  --from-literal=password=YOUR_DATABASE_PASSWORD --from-literal=database=YOUR_DATABASE_NAME
+  --from-literal=host=YOUR_DATABASE_HOST --from-literal=port=YOUR_DATABASE_PORT --from-literal=username=YOUR_DATABASE_USER  --from-literal=password=YOUR_DATABASE_PASSWORD --from-literal=database=YOUR_DATABASE_NAME
```

Comment:
I think those options are weird without the CLI, could we move this to a YAML?

Reply:
They are not Helm values/parameters, they are options used to create the secret so we can't use a yaml at this point, I reworded the sentence and the list of options.